Gordon360/Services/LostAndFoundService.cs

```
@@ -911,5 +911,49 @@ public IEnumerable<FoundItemViewModel> GetFoundItemsAll(string username,
                           action => action.foundID,
                           (foundItem, action) => FoundItemViewModel.From(foundItem, action));
         }
+
+        // Service method to retrieve counts of missing item reports.
+        // It returns a tuple containing the total count of missing items and 
+        // the count after applying the provided filter parameters
+
+        public (int totalCount, int filteredCount) GetMissingItemsCount(
+             string username,
+             string? status,
+             string? color,
+             string? category,
+             string? keywords
+         )
+        {
+            if (!hasFullPermissions(username))
+            {
+                throw new UnauthorizedAccessException();
```

This should actually throw a not found exception (404), since the user doesn't have permission, the API shouldn't even reveal that the route exists.